backend/app/cmd/server_test.go

```diff
@@ -469,7 +469,7 @@ func prepServerApp(t *testing.T, duration time.Duration, fn func(o ServerCommand
 	cmd.Auth.Email.Enable = true
 	cmd.Auth.Email.MsgTemplate = "testdata/email.tmpl"
 	cmd.BackupLocation = "/tmp"
-	cmd.Notify.Type = "telegram"
+	cmd.Notify.Type = []string{"telegram"}
```

Comment:
I don't think you test notify integration path for email in any place. Not sure how to do it because you choose eager init and trying to check smtp right away